### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by erickma1**
Add a mobile menu with the following features
  - When the user clicks (or taps) the hamburger button, the mobile menu appears.
 - When the user clicks (or taps) the close (X) button, the mobile menu disappears.
 - When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
 - When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed

### CODE REVIEW

**Comment by erickma1** [comment]
corrected linter errors

**Review by AmaduKamara — COMMENTED** [review]
# Status: INVALID SUBMISSION ❌ ❌ ❌

Hi @erickma1, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that only you worked on and it's meant to be a pair programming one.

- [ ] Kindly have your partner(s) participate in the project. Except otherwise, and if so, kindly provide proof why only you worked on the project. 👍🏽

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
```

**Comment by erickma1** [comment]
My partner made contributions to the project. Please find below a link to his commit
https://github.com/erickma1/Module1_Portfolio_Setup/pull/9/commits/b5e515c6da1d26e157071152cf1858ce6c8f7352

image

File: style.css [hunk 1]
```diff
@@ -92,6 +92,27 @@ body {
...
+    text-decoration: none;
+    color: #fff;
+    font-family: Roboto, sans-serif;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 133%;
+    margin: 0;
+    padding: 0 16px;
+  }
+
```
Comment (brainconnect93) [inline style.css]: - [ ] Great effort working on your mobile menu, Currently i notice your design is not the same with what we have in the Figma design. I would kindly suggest you make the height of the mobile menu cover the whole page when it is displayed. Also i would want you to look at the font size and be sure they are the same. You can check the id `top-navigation` and give it a `height: 100vh`. I believe this should work for the height issue.

| Figma Design 👇| Your design 👇|
| ------------- | ------------- |
| orig-mob  | mob-men  |

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @erickma1, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## To Highlight 👍

- Descriptive Pull Request title and summary. ✔️
- Has Hamburger Menu functionality ✔️
- Mobile menu options, the correct part of the page is displayed ✔️
- Good use of Github flow ✔️
### Changes Required

- [ ]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by erickma1** [comment]
Add recommended height to the menu
Change the font size to reflect the Figma design

**Review by emmiiorji — APPROVED** [review]
Hi @erickma1 and @ctokoli, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Good Highlights✔️

- Requested changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**